apps.gen.go

```diff
@@ -349,20 +349,20 @@ type AppInstanceSizeCPUType string
 
 // List of AppInstanceSizeCPUType
 const (
-	InstanceSizeCPUType_Unspecified AppInstanceSizeCPUType = "UNSPECIFIED"
-	InstanceSizeCPUType_Shared      AppInstanceSizeCPUType = "SHARED"
-	InstanceSizeCPUType_Dedicated   AppInstanceSizeCPUType = "DEDICATED"
+	AppInstanceSizeCPUType_Unspecified AppInstanceSizeCPUType = "UNSPECIFIED"
+	AppInstanceSizeCPUType_Shared      AppInstanceSizeCPUType = "SHARED"
+	AppInstanceSizeCPUType_Dedicated   AppInstanceSizeCPUType = "DEDICATED"
 )
 
 // DeploymentProgressStep struct for DeploymentProgressStep
 type DeploymentProgressStep struct {
-	Name          string             `json:"name,omitempty"`
-	Status        DeploymentProgressStepStatus `json:"status,omitempty"`
-	Steps         []*DeploymentProgressStep    `json:"steps,omitempty"`
-	StartedAt     time.Time          `json:"started_at,omitempty"`
-	EndedAt       time.Time          `json:"ended_at,omitempty"`
-	Reason        *DeploymentProgressStepReason        `json:"reason,omitempty"`
-	ComponentName string             `json:"component_name,omitempty"`
+	Name          string                        `json:"name,omitempty"`
```

Comment:
Might make sense to throw a `go fmt` in the script that generates these to keep the noise down.